helpers/analytics_helpers.rb

```diff
@@ -0,0 +1,42 @@
+module AnalyticsHelpers
+  def urlize(string)
+    string.downcase.gsub(" ", "_").gsub("-", "_")
+  end
+
+  def build_event(data, attributes = [], output = {})
+    data.sort_by { |k, _v| k["name"] }.each do |item|
+      if item["value"]
```

Comment:
Might be worth pulling out `item["name"]` and `item["value"]` into locals for this block, since they're both referenced a few times.